test/rules/arrayPadding/test.ts.lint

```diff
@@ -0,0 +1,7 @@
+["abc"]
+~~~~~~~ [array padding]
+
+["this", "is", "ok"]
```

Comment:
Ageed.  I don't see the difference.  I would expect:
`[ "this", "is", "ok" ]`
not
`["this", "is", "ok"]`